app/src/main/java/org/piwigo/ui/accounts/ManageAccountsActivity.java

```diff
@@ -0,0 +1,225 @@
+import android.widget.TextView;
+import android.widget.Toast;
+
+import org.piwigo.R;
+import org.piwigo.databinding.ActivityManageAccountsBinding;
+import org.piwigo.ui.login.LoginActivity;
+import org.piwigo.ui.model.User;
+import org.piwigo.ui.shared.BaseActivity;
+import org.piwigo.ui.view.AccountView;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+import dagger.android.AndroidInjection;
+
+public class ManageAccountsActivity extends BaseActivity implements AccountView {
+    @Inject ManageAccountsViewModelFactory viewModelFactory;
+
+    /* TODO: cleanup activity:
+     *   selection by tap is crazy, tap shall directly open the LoginView
+     *   LongClick could delete after confirmation
+     *   currently active account should be highlighted
+     *   */
+    ManageAccountsViewModel viewModel;
+    ActivityManageAccountsBinding binding;
+
+    private ListView userListView;
+
+    private UserAdapter mUserAdapter;
+
+
+    private class UserAdapter extends ArrayAdapter<User>{
+        public UserAdapter(List<User> objects) {
+            super(ManageAccountsActivity.this, 0, objects);
+        }
+        @Override
+        public View getView(int position, View view, ViewGroup parent) {
+            final User user = getItem(position);
+            if (view == null) {
+                LayoutInflater inflater = ManageAccountsActivity.this.getLayoutInflater();
+                view = inflater.inflate(R.layout.account_row, parent, false);
+            }
+            TextView username = (TextView) view.findViewById(R.id.username);
+            username.setText(user.username);
+            if(user.guest) {
+                username.setTypeface(Typeface.DEFAULT, Typeface.BOLD_ITALIC);
+            }else{
+                username.setTypeface(Typeface.DEFAULT, Typeface.BOLD);
+            }
+
+            TextView galleryUrlView = (TextView) view.findViewById(R.id.gallery_url);
+            galleryUrlView.setText(user.url);
+
+            ImageView imageView = (ImageView) view.findViewById(R.id.account_image);
+/* TODO fill image here (from favicon of gallery / User) */
+
+            return view;
+        }
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        AndroidInjection.inject(this);
+        setTheme(R.style.Theme_Piwigo);
+        super.onCreate(savedInstanceState);
+
+        viewModel = ViewModelProviders.of(this, viewModelFactory).get(ManageAccountsViewModel.class);
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_manage_accounts);
+        binding.setViewModel(viewModel);
+
+        /* TODO: isn't it bulls*** to set the title of the viewmodel from the activity and wouldn't this be better handled in the ViewModel directly? */
+        viewModel.title.set(getString(R.string.nav_manage_accounts));
+        viewModel.users.set(accountHelper.getUsers());
+        viewModel.setView(this);
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.account_toolbar);
```

Comment:
You don't need to use `findViewById` as the binding already has a reference to any view with an id

Reply:
can you point me to an example?

Reply:
The binding class will already have a reference e.g. `binding.accountToolbar`